src/demo/DocsDecorator/DocsDecorator.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import block from 'bem-cn-lite';
+import {DocsContainer, DocsContainerProps} from '@storybook/addon-docs/blocks';
+
+import {ThemeProvider, MobileProvider} from '../..';
+
+import './DocsDecorator.scss';
+
+export interface DocsDecoratorProps extends React.PropsWithChildren<DocsContainerProps> {}
+
+const b = block('docs-decorator');
+
+export function DocsDecorator({children, context}: DocsDecoratorProps) {
+    return (
+        <div className={b()}>
+            <DocsContainer context={context}>
+                <ThemeProvider theme="light">
```

Comment:
Theme is hard-coded? Shouldn't this value be taken from the storybook theme knob?